Writing-editing network/main.py

```diff
@@ -183,7 +192,8 @@ def train_generator(input_variable, input_lengths, target_variable, topics, mode
         # this is not the eval mode.
         if not is_eval:
             """ Call Discriminator, Critic and get the ReINFORCE Loss Term"""
-            reinforce_loss = None
+            est_values = critic_model(input)
+            reinforce_loss = reinforce(gen_log, dis_out, est_values, seq_length, CommonConfig())
```

Comment:
`gen_log` is I think the generator's log probabilities. What is `dis_out`? If it is the output of the discriminator, then I cannot see a call to the discriminator here that would get the output. Please add that.